AUTHORS.txt

```diff
@@ -0,0 +1,9 @@
+# This is the official list of project authors for copyright purposes.
+# This file is distinct from the CONTRIBUTORS.txt file.
+# See the latter for an explanation.
+# 
+# Names should be added to this file as:
+# Name or Organization <email address>
+
+Deni Anggara <>
+Github <>
```

Comment:
@tobiasahlin all authors and copyright holders should be listed here, ideally with a email address to contact them